core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -50,8 +50,10 @@
     </div>
   </div>
   <div class="oppia-opportunities-button-container oppia-animate-fadein">
-    <button class="oppia-opportunities-button protractor-test-opportunity-list-item-button"
+    <button class="protractor-test-opportunity-list-item-button"
```

Comment:
How would user know why this button is disabled?

Reply:
User may assume thae button is disabled since the progress bar is completed and once the user hover over the percentage, it says No cards available.

Reply:
The progress will not be 100% considering the suggestions are in review (not accepted yet), right?

Reply:
@DubeySandeep Three colurs were introduced to the translation progress bar here. If all are in review, the progressbar will be 100% in light green colour. Reference - https://github.com/oppia/oppia/pull/13835

Reply:
Can we also add a tooltip to the translate button on hover explaining why it's disabled? e.g.

"All available translations are currently in review."

Reply:
Done